src/contextExplorer.ts

```diff
@@ -0,0 +1,211 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as fs from 'fs';
+import * as path from 'path';
+import {config} from 'process';
+import {stringify} from 'querystring';
+import * as vscode from 'vscode';
+import { Model } from './Circlereader/circle-analysis/circle/model';
+
+enum NodeType{
+  directory,
+  model,
+  config,
+  configWrapper
```

Comment:
(Q) Do we need config wrapper directory? How about showing `cfg` file directly without a config wrapper director under the model file?

Reply:
Sure. @hyunsik-yoon game me the same opinion, I have removed the wrapper node.